src/metric/populate_test.go

```diff
@@ -143,13 +155,34 @@ func TestPopulateK8s(t *testing.T) {
 	}
 
 	k8sVersion := &version.Info{GitVersion: "v1.15.42"}
-	err = p.Populate(foo, kubeletSpecs, i, "test-cluster", k8sVersion)
+	err = p.Populate(foo, kubeletSpecs, intgr, "test-cluster", k8sVersion)
 	require.IsType(t, err, data.PopulateResult{})
 	assert.Empty(t, err.(data.PopulateResult).Errors)
 
-	expectedInventory := sdk.Inventory{}
-	expectedInventory.SetItem("cluster", "name", expectedMetrics[0].Entity.Name)
-	expectedInventory.SetItem("cluster", "k8sVersion", k8sVersion.String())
-	expectedMetrics[0].Inventory = expectedInventory
-	assert.ElementsMatch(t, expectedMetrics, i.Data)
+	expectedInventory := inventory.New()
+	err = expectedInventory.SetItem("cluster", "name", expectedEntities[0].Metadata.Name)
+	require.NoError(t, err)
+
+	err = expectedInventory.SetItem("cluster", "k8sVersion", k8sVersion.String())
+	require.NoError(t, err)
+	expectedEntities[0].Inventory = expectedInventory
```

Comment:
nit: making use of whitespace allow to separate group of operations, which enhances readability.
```suggestion
	expectedInventory := inventory.New()
	
	err = expectedInventory.SetItem("cluster", "name", expectedEntities[0].Metadata.Name)
	require.NoError(t, err)

	err = expectedInventory.SetItem("cluster", "k8sVersion", k8sVersion.String())
	require.NoError(t, err)
	
	expectedEntities[0].Inventory = expectedInventory
```